components/popover/popover-mixin.js

```diff
@@ -53,14 +73,56 @@ export const PopoverMixin = superclass => class extends superclass {
 			:host([_opened]) {
 				display: inline-block;
 			}
+			:host([_location="block-start"]) {
```

Comment:
These logical positioning terms are being used to align more closely with what the browser is doing, as well as to set us up for supporting other locations.
`block-start` -> top (this PR)
`block-end` -> bottom (this PR)
`inline-start` -> left (future PR)
`inline-end` -> right (future PR)